articles/quickstarts/index.md

```diff
@@ -19,17 +19,144 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
 - APIs for Python and .NET languages (C#, F#, and VB.NET) for running quantum programs written in Q#
 - Tools to facilitate your development
 
-Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, or through Jupyter Notebooks with the IQ# Jupyter kernel.
-They can also be paired with a host program written in a .NET language (typically C#) or Python, enabling you to call quantum operations from inside a classical program.
+Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, through Jupyter Notebooks with the IQ# Jupyter kernel, or paired with a host program written in a .NET language (typically C#) or Python. You can also run Q# programs online using [Codespaces](https://online.visualstudio.com/) or [MyBinder.org](https://mybinder.org/). 
 
-The workflows for each of these setups are described and compared at [Ways to run a Q# program](xref:microsoft.quantum.guide.host-programs).
+## Installation options
+
+You can use the QDK three ways:
+
+- [Install the QDK locally](#install-the-qdk-locally)
+- [Use the QDK online](#use-the-qdk-online)
+- [Use a QDK Docker image](#use-the-qdk-with-docker)
+
+## Install the QDK locally
+
+You can develop Q# code in most of your favorites IDEs, as well as integrate Q# with other languages such as Python and .NET (C#, F#).
+
+|&nbsp; | **VS Code<br>(2019 or later)**| **Visual Studio<br>(2019 or later)** | **Jupyter Notebooks** | **Command line**|
```

Comment:
"Command line" is a column here, but I don't see any corresponding instructions below? What is this supposed to indicate to the user?

Reply:
Since the command line functionality overlaps with other install scenarios, should the headers of the sections be updated, for example "Configure for VS Code and command line"?

Reply:
@cgranade if you only wanted to use the command line, how would you install the QDK?
Wouldn't you need to install one of the extensions or the Q# kernel?

Reply:
Not necessarily. The extensions provide design-time functionality for VS Code and VS 2019, while the kernel provides interoperability with Jupyter Notebook and with Python. One can definitely build and run Q# programs from the command-line using _only_ the .NET Core SDK itself and a text editor (some of our users prefer vim, for instance). That said, I definitely agree that it's going to be significantly easier to have extensions for design-time support.

Reply:
@cgranade So the basic requirements will only be to install .NET Core SDK to use the command line option here?
If so @Bradben, you can add the missing instructions for the commandline option

Reply:
@efratshabtai: That's correct for writing and running Q# standalone or Q# + .NET applications at the command-line, yes.